camviewer_ui_impl.py

```diff
@@ -665,6 +666,24 @@ def closeEvent(self, event):
             self.dumpConfig()
         QMainWindow.closeEvent(self, event)
 
+    def end_monitors(self):
+        all_mons = []
+        all_mons.extend(self.camconn_pvs)
+        all_mons.extend(self.globmarkpvs)
+        all_mons.extend(self.globmarkpvs2)
+        all_mons.extend(self.otherpvs)
+        all_mons.append(self.notify)
+        all_mons.append(self.rowPv)
+        all_mons.append(self.colPv)
+        all_mons.append(self.launch_gui_pv)
+        all_mons.append(self.launch_edm_pv)
+        all_mons.append(self.lensPv)
+        all_mons.append(self.calibPV)
+        for pv in all_mons:
+            if pv is not None:
```

Comment:
do pyca pvs delete themselves and replace themselves with `None` when they're disconnected?

Reply:
This is a camviewer special where it stuffs `None` as a stand-in for these attributes when they fail to connect